apis/networkconnectivity/v1alpha1/serviceconnectivity.go

```diff
@@ -0,0 +1,213 @@
+	   ComputeNetwork's current state. */
+	Conditions []v1alpha1.Condition `json:"conditions,omitempty"`
+
+	/* ObservedGeneration is the generation of the resource that was most recently observed by the Config Connector controller. If this is equal to metadata.generation, then that means that the current reported status reflects the most recent desired state of the resource. */
+	// +optional
+	ObservedGeneration *int `json:"observedGeneration,omitempty"`
+
+	ObservedState *ServiceConnectionPolicyObservedState `json:"observedState,omitempty"`
+}
+
+type ServiceConnectionPolicyObservedState struct {
+	// // CreateTime: Output only. Time when the ServiceConnectionMap was
+	// // created.
+	// CreateTime string `json:"createTime,omitempty"`
+
+	// // Etag: Optional. The etag is computed by the server, and may be sent
+	// // on update and delete requests to ensure the client has an up-to-date
+	// // value before proceeding.
+	// Etag string `json:"etag,omitempty"`
+
+	// Infrastructure: Output only. The type of underlying resources used to
+	// create the connection.
+	//
+	// Possible values:
+	//   "INFRASTRUCTURE_UNSPECIFIED" - An invalid infrastructure as the
+	// default case.
+	//   "PSC" - Private Service Connect is used for connections.
+	Infrastructure string `json:"infrastructure,omitempty"`
+
+	// // Name: Immutable. The name of a ServiceConnectionPolicy. Format:
+	// // projects/{project}/locations/{location}/serviceConnectionPolicies/{ser
+	// // vice_connection_policy} See:
+	// // https://google.aip.dev/122#fields-representing-resource-names
+	// Name string `json:"name,omitempty"`
+
+	// PscConnections: Output only. [Output only] Information about each
+	// Private Service Connect connection.
+	PscConnections []*PscConnection `json:"pscConnections,omitempty"`
+
+	// // UpdateTime: Output only. Time when the ServiceConnectionMap was
+	// // updated.
+	// UpdateTime string `json:"updateTime,omitempty"`
+
+}
+
+type PscConfig struct {
+	// Limit: Optional. Max number of PSC connections for this policy.
+	Limit *int64 `json:"limit,omitempty,string"`
+
+	// Subnetworks: The resource paths of subnetworks to use for IP address
+	// management. Example:
+	// projects/{projectNumOrId}/regions/{region}/subnetworks/{resourceId}.
+	Subnetworks []string `json:"subnetworks,omitempty"`
+}
+
+type PscConnection struct {
+	// ConsumerAddress: The resource reference of the consumer address.
+	ConsumerAddress *string `json:"consumerAddress,omitempty"`
+
+	// ConsumerForwardingRule: The resource reference of the PSC Forwarding
+	// Rule within the consumer VPC.
+	ConsumerForwardingRule *string `json:"consumerForwardingRule,omitempty"`
+
+	// ConsumerTargetProject: The project where the PSC connection is
+	// created.
+	ConsumerTargetProject *string `json:"consumerTargetProject,omitempty"`
+
+	// // Error: The most recent error during operating this connection.
+	// Error *GoogleRpcStatus `json:"error,omitempty"`
```

Comment:
Should we have something which either tracks or explains why these fields are commented out?  If because its deprecated maybe we should track that?